components/blocks/subNewsLettersButton.tsx

```diff
@@ -14,11 +14,18 @@ export const SubNewsLettersButton = ({ headerText, subscribeButtonText }) => {
   const [lastName, setLastName] = useState("");
   const [infoMessage, setInfoMessage] = useState("");
   const [isContactExisting, setIsContactExisting] = useState(false);
+  const [allInputsFilled, setAllInputsFilled] = useState(false);
   const [isLoading, setIsLoading] = useState(false);
 
   const handleSubscribe = async () => {
-    if (email === "" || firstName === "" || lastName === "") return;
+    setInfoMessage("");
+    if (email === "" || firstName === "" || lastName === "") {
```

Comment:
We use Formik on the booking form, maybe it could be used here? (it might be overkill)

See how its used here:  https://github.com/SSWConsulting/SSW.Website-v3/blob/c6b39b515ce3686b208200435777abe65e4fe3d6/components/bookingForm/bookingForm.tsx#L154-L165